support/docker/production/docker-compose.yml

```diff
@@ -30,16 +30,11 @@ services:
     image: chocobozzz/peertube:production-buster
     env_file:
       - .env
-    # Traefik labels are suggested as an example for people using Traefik,
-    # remove them if you are using another reverse proxy.
-    labels:
-      traefik.enable: "true"
-      traefik.frontend.rule: "Host:${PEERTUBE_WEBSERVER_HOSTNAME}"
-      traefik.port: "9000"
     # If you don't want to use a reverse proxy (not suitable for production!)
     # ports:
       # - "80:9000"
     volumes:
+      - ./docker-volume/assets:/client/dist
```

Comment:
Any reason why this new volume is needed?

Reply:
Yes, to allow this block to be functional: https://github.com/Chocobozzz/PeerTube/blob/develop/support/nginx/peertube#L62-L67

Reply:
But I made a mistake: it should be bound to `/app/client/dist`.

Reply:
Ah my nginx config looks pretty differently, I just cache all the frontend files.

https://yerbamate.dev/nutomic/peertube-compose/src/branch/master/templates/nginx.conf

Reply:
I didn't think about that. I'll try this.